### PR TITLE
Add a section on pair programming

### PR DESCRIPTION
**Pull request by stacietaylorcima**
## Description 

This PR adds a section to the Project Brief on Pair Programming. This is a very important part of Collab Lab and adding some more detail to the Project Brief will help members launch into the project with more confidence. 

## Goals

There were three important things I want to convey here so that our team knows what to expect.

1. The idea that everyone is expected to pair live with their partner during the week. I teetered on the idea of saying something like "at least one time per week" but decided that didn't seem totally necessary. Thoughts here?
2. Why pair programming is great and everyone should be excited for it!
3. A high-level of what pair programming looks like at Collab Lab.

Let me know what you think here! 🤗

### CODE REVIEW

File: PROJECT-BRIEF.md [hunk 1]
```diff
@@ -52,6 +52,14 @@ Pairings will go as follows:
 1. {Name} & {Name}
 2. {Name} & {Name}
 
+### Pair Programming
+
+A portion of your dedicated hours each week will be spent [pair programming](https://www.freecodecamp.org/news/how-remote-pair-programming-works-and-why-it-can-change-your-life-cd7b767dc60f/) with your partner -- this is where the most magical collaboration happens.
```
Comment (stacietaylorcima) [inline PROJECT-BRIEF.md]: This isn't an article that I had a strong pull towards, it just seemed like the most digestible, most on-brand one I found after a short search for articles on pair programming. LMK if you have good resources here. I def wanna link to somewhere they can learn more.

Comment (segdeha) [reply PROJECT-BRIEF.md]: That looks good enough to use to start with to me! It covers the main points such as drive/navigator and is easy to scan and digest

File: PROJECT-BRIEF.md [hunk 2]
```diff
@@ -52,6 +52,14 @@ Pairings will go as follows:
...
+
+Pair programming offers some really exciting opportunities to teach/learn from your partner, get a different perspective and/or insight on the issues you're facing, and drive forward progress on your goals.
+
+There is no one "right" way to pair program, but in general, you will meet with your partner on Zoom to work through the story or task you've chosen for that week. You're encouraged to test out some different approaches to pair programming to see what works best for your and your partner's collaboration styles.
```
Comment (segdeha) [inline PROJECT-BRIEF.md]: I’m a stickler for using the correct quote marks, so I prefer `“right”` to `"right"` and `you’re` to `you're`. Thanks for indulging my pedantic-ness!

**Review by segdeha — CHANGES_REQUESTED** [review]
Looks great! My only requested changes are around using proper/curly/smart quote marks. Thanks for doing this!

File: PROJECT-BRIEF.md (hunk 1 above)
Comment (segdeha) [inline PROJECT-BRIEF.md]: Oh, whoops, should have caught this the first time. Can you please replace ` -- ` with `—` ← that’s an “em” dash, which is the proper typographic mark for delineating a section of a sentence like this one.

Comment (segdeha) [reply PROJECT-BRIEF.md]: (I realize this level of attention to detail is overkill for a README, but it’s the kind of thing I can’t help but notice.) 😬

File: PROJECT-BRIEF.md [hunk 3]
```diff
@@ -52,6 +52,14 @@ Pairings will go as follows:
 1. {Name} & {Name}
 2. {Name} & {Name}
 
+### Pair Programming
+
+A portion of your dedicated hours each week will be spent [pair programming](https://www.freecodecamp.org/news/how-remote-pair-programming-works-and-why-it-can-change-your-life-cd7b767dc60f/) with your partner -- this is where the most magical collaboration happens.
+
+Pair programming offers some really exciting opportunities to teach/learn from your partner, get a different perspective and/or insight on the issues you’re facing, and drive forward progress on your goals.
+
+There is no one “right” way to pair program, but in general, you will meet with your partner on Zoom to work through the story or task you’ve chosen for that week. You’re encouraged to test out some different approaches to pair programming to see what works best for your and your partner’s collaboration styles.
```
Comment (segdeha) [inline PROJECT-BRIEF.md:61]: Lovely! 👏🏻👏🏻👏🏻

File: PROJECT-BRIEF.md [hunk 4]
```diff
@@ -97,11 +105,11 @@ When you and your pair partner have working code that you believe is ready to be
...
+5. Once approved, merge the PR into `master`. (Your code will be built and deployed to production automatically thanks to [Netlify](https://www.netlify.com/).)
 6. Check your work on the [production site]({link to cohort live site}).
 7. Celebrate! 🥳
 
 ### What else?
 I’m around 💯% sure I’m forgetting or glossing over something important, so please be noisy on Slack as things come up. The mentors and I will do our best to get you unstuck. Also, lean on each other for help as well! Finally, work in the open so everyone can benefit from your questions.
 
-![gif of a My Little Pony pulling on goggles, with a determined look on their face, saying, "Let's do this".](http://giphygifs.s3.amazonaws.com/media/PuWNMebKGIKNG/giphy.gif)
\ No newline at end of file
+![gif of a My Little Pony pulling on goggles, with a determined look on their face, saying, “Let’s do this”.](http://giphygifs.s3.amazonaws.com/media/PuWNMebKGIKNG/giphy.gif)
```
Comment (segdeha) [inline PROJECT-BRIEF.md:115]: Niiiiiiice!!! 😍🤩🥳

**Review by segdeha — DISMISSED** [review]
Just one little change that I missed the first time through. Thank you!

**Pull request merged**